tests/testBackground.py

```diff
@@ -513,22 +458,8 @@ def testOnlyOneGridCell(self):
                 testval = afwMath.cast_BackgroundMI(backobj).getPixel(bctrl.getInterpStyle(), xpix, ypix)
                 self.assertAlmostEqual(testval/mean, 1)
 
-    def testTicket1681OffByOne(self):
```

Comment:
That's not correct. You don't need an assert to have a test. If the test code crashes then the test fails. Some tests really do exist purely to ensure that nothing breaks even if nothing is tested explicitly.